### PR TITLE
Update documentation for simple sample run

### PR DESCRIPTION
**Pull request by psx95**
### Issue - Instructions to run 'Simple' sample needed a little more clarity

 - Following the instructions to create a GKE cluster and run OTel collector needed a little more clarity from the perspective of a new user with less knowledge of kubernetes. 
 - What was the expected outcome as a result of following this sample as-is was also unclear. 

### Fix 
 - This PR attempts to add more clarity to the instructions for running the sample by being more explicit.  

### Testing 
 - Was able to re-run the sample with relative ease following these instructions (Started from a clean-slate).

### CODE REVIEW

File: deploy/gke/setup.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+* A built and pushed collector container image.  See [Building a Collector Image Locally](../../build/local/README.md) for instructions for a local build.
+* A GCP project with billing enabled.
 * A running GKE cluster
-* Artifact Registry enabled in your GCP project
-* Cloud Metrics, Cloud Trace, and/or Cloud Logging APIs enabled (optional depending on exporters configured in your collector)
+    - GKE offers 2 kinds of clusters - [Autopilot](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-an-autopilot-cluster) and [Standard](https://cloud.google.com/kubernetes-engine/docs/concepts/regional-clusters). These sample should work with both kinds of clusters. 
+    - After creating the clusters, you will have to connect your local command line to the created cluster in order to interact with it. The steps to do this are available on your cloud console. 
+        - On Google Cloud Console, locate & click the cluster you just created, on the resulting details page, there will be a `Connect` button on the top of the page containing the instructions to configure the cluster for local command line access. *(You should have `kubectl` installed for this)*
```
Comment (damemi) [inline deploy/gke/setup.md]: these steps are more related to GKE than our project, so I don't think we need them. GCP beginners should be able to find these steps in the GCP docs

File: deploy/gke/simple/README.md [hunk 2]
```diff
@@ -3,7 +3,11 @@
 If this is the first example you are trying out, follow the [Setup](../setup.md) instructions to
 complete the prerequisites.
 
-See the [Troubleshooting](../troubleshooting.md) guide for more information if you encounter issues.
+**NOTE:** As a reminder, if you completed the pre-requisite steps successfully, the following requirements should have already been met - 
+ - You should see the image name in [manifest.yaml](./manifest.yaml) updated with the fully qualified docker image name. 
+ - The above mentioned docker image is actually uploaded to Artifact Registry and is visible there in cloud console.
+
+If the above requirements are not met, please ensure that all the [Setup](../setup.md) instructions have been followed. You may need to perform a [local build](../../../build/local/README.md) again.
```
Comment (damemi) [inline deploy/gke/simple/README.md:10]: this feels redundant to the links that are in the setup doc

File: deploy/gke/simple/README.md [hunk 3]
```diff
@@ -34,6 +38,38 @@ Create this manifest in your cluster with:
...
+
+ There are many metrics that are emitted from the OpenTelemetry collector itself and most of these metrics start with the prefix `Otelcol` and you can search for this string in metrics explorer. 
+
+#### Potential Issues 
+If you are not seeing metrics on Google Cloud, first run `kubectl logs <pod_name> -n $OTEL_NAMESPACE` to verify that they are being printed locally within the pod container and if any error/issue is being reported while exporting metrics to Google Cloud. 
+
+Most likely, this issue is due to permission errors. Try following the troubleshooting steps mentioned here - [Permission errors exporting telemetry](../troubleshooting.md#permission-errors-exporting-telemetry).
+
+*NOTE: You can chnage the configuration for the OpenTelemetry Collector to alter its behaviour.*
+
```
Comment (damemi) [inline deploy/gke/simple/README.md]: this section could be moved to the troubleshooting doc

File: deploy/gke/troubleshooting.md [hunk 4]
```diff
@@ -14,11 +14,13 @@ to identify it as the Workload Identity account.
 If you don't already have one, create a GCP service account to authorize the collector to send metrics, traces, and logs:
 
 ```
-export GCLOUD_PROJECT=<your GCP project>
+export GCLOUD_PROJECT=<the project ID of the Google Cloud project of your IAM service account>
```
Comment (damemi) [inline deploy/gke/troubleshooting.md]: `the project ID of the Google Cloud project of your IAM service account`  maybe rephrase to be clearer?

File: deploy/gke/setup.md (hunk 1 above)
Comment (punya) [reply deploy/gke/setup.md]: One way to handle this would be to add a "pre-requisites" note before the instructions, saying that we expect a reader to already know how to set up a GKE cluster and connect to it from a local environment. We can provide a single link to a GKE getting started guide rather than duplicating the content piecemeal.

File: deploy/gke/troubleshooting.md (hunk 4 above)
Comment (psx95) [reply deploy/gke/troubleshooting.md]: The current one is taken directly from the GKE documentation but how about - 
```
The Google cloud project ID in which your IAM service account resides 
```
I don't have a strong feeling on this, so open to suggestions here.

File: deploy/gke/simple/README.md (hunk 2 above)
Comment (psx95) [reply deploy/gke/simple/README.md:10]: So while trying the steps from this sample, treating `build` and `gke` as 2 different/unrelated things and doing them separately what I failed to realize was that running the local build actually modifies the  `manifest.yaml` file. 

Also, the local build will have to be run for both samples independently (since it only modifies 1 `manifest.yaml` per build). 
So I believe while a little redundant, having this section here makes the process a little more transparent and easier to follow.

File: deploy/gke/setup.md (hunk 1 above)
Comment (psx95) [reply deploy/gke/setup.md]: I see, I can update this.

File: deploy/gke/simple/README.md (hunk 3 above)
Comment (psx95) [reply deploy/gke/simple/README.md]: I can update and move it to troubleshooting doc.

**Review by damemi — APPROVED** [review]
